caddyhttp/browse/browse_test.go

```diff
@@ -300,7 +300,7 @@ func TestBrowseJson(t *testing.T) {
 		req.Header.Set("Accept", "application/json")
 		rec := httptest.NewRecorder()
 
-		code, err := b.ServeHTTP(rec, req)
+		code, _ := b.ServeHTTP(rec, req)
```

Comment:
@elcore Of course, but I use it too and don't see that output. O.o

And the CI tests have been running it too for the last several months without showing it.